### ISSUE TITLE
an ode from a mildy inconvenienced sodapop (Victini Event Reward Issue)

### ISSUE THREAD

**Issue by OrangeSodaPop** [ObliqueNET/Server#1192]
Hello my name is orangesodapop and I have now figured out that my event victini was given out improperly after using 1 nature voucher and a lot of iv vouchers (I don't remember how many, but I know I used at least 30). This is because after speaking to Project (Dogi) I was told he got a 5iv  with 0 attack victini with a z celebrate set, which is normally impossible in pixelmon as celebrate cant be taught to victini by tutors. However, we both were supposed to get the same reward according to Junichi
![issue](https://user-images.githubusercontent.com/57567713/136116899-bd5335a6-247f-478b-a907-e3a195b19353.png)
![issue 2](https://user-images.githubusercontent.com/57567713/136116920-1ce4dcd7-50c0-4870-b197-b925814cde9a.png)

so basically I am asking if I can get a refund on the victini to make it a z celebrate set, as well as the vouchers I used on it (if there's a way to figure out the exact amount of vouchers I used on it that would be pretty cool)

anyways thank you for your time, orange out, if anyone has any more inquiries, send me a dm at OrangeSoda#1769

have a pufferfish as a condolence
![shutterstock_83594650](https://user-images.githubusercontent.com/57567713/136117296-57c9a2b5-bb6c-47bc-8499-16983da77383.jpg)

**Comment by Katadreme** [comment]
Thanks for the fish. Has been refunded so am closing.
